pkg/columns/columns.go

```diff
@@ -322,3 +324,8 @@ func (c *Columns[T]) MustSetExtractor(columnName string, extractor func(*T) stri
 		panic(fmt.Errorf("setting extractor for %q column: %w", columnName, err))
 	}
 }
+
+// GetField is a helper to retrieve a value of type OT from a struct of type T given an offset
+func GetField[OT any, T any](entry *T, offset uintptr) OT {
+	return *(*OT)(unsafe.Add(unsafe.Pointer(entry), offset))
```

Comment:
As far as I understand it, it wouldn't make any difference if we would split the arithmetic there - as the entry always comes from outside the function and won't be GC'ed until at least the function returns. Since we're doing all of this always inside of functions which have the entry as parameter and we never store/use anything (and especially not a uintptr) outside of these functions, we shouldn't run into any trouble regarding the GC, right?

But I'll add the note just to be safe.